core/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -363,6 +367,29 @@ class ApplicationInfoSuite extends FunSuite with Logging {
     }
   }
 
+  test("test read GPU datasource- IoMetrics") {
+    TrampolineUtil.withTempDir { tempOutputDir =>
+      var apps: ArrayBuffer[ApplicationInfo] = ArrayBuffer[ApplicationInfo]()
+      val appArgs = new ProfileArgs(Array(s"$logDir/eventlog-gpu-dsv1.zstd"))
+      var index: Int = 1
+      val eventlogPaths = appArgs.eventlog()
+      for (path <- eventlogPaths) {
+        apps += new ApplicationInfo(hadoopConf,
+          EventLogPathProcessor.getEventLogInfo(path,
+            sparkSession.sparkContext.hadoopConfiguration).head._1, index)
+        index += 1
+      }
+      assert(apps.size == 1)
+      val analysis = new Analysis(apps)
+      val ioMetrics = analysis.ioAnalysis()
+      assert(ioMetrics.size == 5)
+      val metricsSqlId1 = ioMetrics.filter(metrics => metrics.sqlId == 1)
+      assert(metricsSqlId1.size == 1)
+      assert(metricsSqlId1.head.inputBytesReadSum == 1348)
+      assert(metricsSqlId1.head.inputRecordsReadSum == 66)
```

Comment:
can we test all fields?

Reply:
done.  Most of the values are 0 i.e not written, no disk spilling etc. 
PTAL.